src/containers/Chat/ChatMessages/ListReplyTemplate/ListReplyTemplate.tsx

```diff
@@ -158,12 +160,27 @@ export const ListReplyTemplateDrawer: React.SFC<ListTemplate> = (props) => {
         <div className={styles.Options}>
           {options
             .map((option: any) => {
+              const payloadObject = {
+                payload: {
+                  type: 'list_reply',
+                  title: option.title,
+                  id: '',
```

Comment:
Should we be generating unique id here?

Reply:
The id comes empty from the gupshup end as well. Since we are making it exactly same let's not add it for now. If needed we can add it later. 

We can also discuss this in our call.

Reply:
ok